cpp/src/slice2cpp/Gen.cpp

```diff
@@ -1787,10 +1815,21 @@ Slice::Gen::ProxyVisitor::visitOperation(const OperationPtr& p)
     H << sp;
     if (comment)
     {
-        StringList postParams, returns;
-        postParams.push_back(contextDoc);
-        returns.push_back(futureDoc);
-        writeOpDocSummary(H, p, *comment, OpDocInParams, false, {}, StringList{}, postParams, returns);
+        StringList postParamsDoc;
+        postParamsDoc.push_back(contextDoc);
+
+        StringList futureDoc;
+        if (futureOutParams.size() == 0)
```

Comment:
Why not just:
```suggestion
        if (futureOutParams.empty())
```
And if so, I'm surprised the linter doesn't complain about this.